engine/apps/integrations/mixins/alert_channel_defining_mixin.py

```diff
@@ -66,13 +73,6 @@ def dispatch(self, *args, **kwargs):
                 logger.info("Cache is empty!")
                 raise
 
-        if alert_receive_channel.organization.is_moved:
-            raise OrganizationMovedException(alert_receive_channel.organization)
-        if alert_receive_channel.organization.deleted_at:
-            # It's better to raise OrganizarionDeletedException, but in legacy code PermissionDenied is returned when integration key not found.
-            # So, keep it consistent.
-            raise PermissionDenied("Integration key was not found. Permission denied.")
-
```

Comment:
I had to move this. if we try accessing the `alert_receive_channel.organization` while the database is down, this tries to make a call to the db to fetch the org, and obviously fails

Reply:
This could also be a part of an `else` block of the `try..except`, right? Just to avoid making the `try` block bigger and catching potential exceptions here?